### PR TITLE
Create invite-template.md

### PR DESCRIPTION
**Pull request by GeorgiaHCA**
This PR creates a new folder to hold information about the fortnightly meet-up sessions. 

It would be useful to know from reviewers:

* If the agenda seems sensible
* If the subgroups make sense (they have been co-created with previous attendees)
* Any improvements to the markdown file itself

Thanks! 

(I've added a lot of reviewers I think would be best able to help - but if you're too busy don't worry!)

### CODE REVIEW

File: project-management/engagement-guides/meet-up-sessions/invite-template.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+* A moderator can help you choose a group to join or an activity to take part in based on your skills and interests if you are unsure.
+* Join back here for group discussion
+
+------
+
+## Explore Current Activities in AutSPACEs
+
+If you're wondering where to start, check out some of the current activities that people are leading. You are welcome to join them, or propose a new task. 
+
+### Comms, engagement, and promotion
```
Comment (EKaroune) [inline project-management/engagement-guides/meet-up-sessions/invite-template.md:80]: Don't use Comms - change to communications to be more clear

File: project-management/engagement-guides/meet-up-sessions/invite-template.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+*Suggested activities*  
+
+**Sign up below** 
+*Please add your name below if you are or would like to be involved in these activities (completely optional!)*
```
Comment (EKaroune) [inline project-management/engagement-guides/meet-up-sessions/invite-template.md:85]: I think you should add at least one * (bullet point) below to start the list of names for each of the groups and then others can copy this tp add more. People might not be familiar with using hackmd.

**Review by EKaroune — APPROVED** [review]
Looks good, well done - this will make your meeting more consistent and help to onboard people. You might consider adding in a section on how to use hackmd? (could just be a link to a help document), although I see you also have google docs to use.

File: project-management/engagement-guides/meet-up-sessions/invite-template.md (hunk 2 above)
Comment (GeorgiaHCA) [reply project-management/engagement-guides/meet-up-sessions/invite-template.md:85]: Good idea Emma 💛

**Comment by GeorgiaHCA** [comment]
Thanks Emma! I've included your changes. Mostly I use Google Docs as more people are familiar. Merging away!

**Review by EKaroune — APPROVED** [review]
Look good. 😄

**Pull request merged**